src/datumaro/plugins/data_formats/ade20k2020.py

```diff
@@ -111,10 +111,11 @@ def _load_items(self, subset):
                 for class_idx, label_name in classes:
                     label_id = labels.find(label_name)[0]
                     item_annotations.append(
-                        Mask(
+                        ExtractedMask(
+                            index_mask=mask,
+                            index=class_idx,
```

Comment:
shouldn't this be `index=label_id`? `class_idx` can be different to `label_id` because of `label_categories` generation in Datumaro.

Reply:
This should be 100% equivalent to the previous implementation (`mask.lazy_extract(class_idx)`). The previous implementation extract a binary mask with `mask.lazy_extract(class_idx)`. Then it can use it with its `label=label_id`. This is exactly same as the current implementation. It just replace `mask.lazy_extract(class_idx)` with `ExtractedMask(index_mask=mask, index=class_idx, ...)`. The only difference is that we explicitly maintain the source reference `index_mask` as ExtractedMask.index_mask`.